src/test/java/use_case/multiplayer/MultiPlayerInteractorTest.java

```diff
@@ -87,6 +111,123 @@ void advancingPastDeckEndsGame() {
     assertEquals("Deck complete", state.getMessage());
   }
 
+  @Test
+  void chooseAnswerIgnoredWhenGameIsOver() {
+    interactor.startGame(firstDeck(), gateway.getSession().getUser(), new User("guest", "guest"));
+    interactor.advance();
+    interactor.advance();
+    interactor.advance(); // triggers end
```

Comment:
```suggestion
   for(int i = 0; i < firstDeck().getCardCount(); i++) {
     interactor.advance();
   }
```
instead of relying on the fixed count of cards, the code should have more flexibility so that testing with different decks will not cause an issue, and also for reducing the length of the code.